FILE_TEMPLATE

```diff
@@ -1,3 +1,3 @@
-// Copyright 2019-2023 Parity Technologies (UK) Ltd.
+// Copyright 2019-2025 Parity Technologies (UK) Ltd.
```

Comment:
dq: Maybe we can remove the years entirely? And so a similar update to polkadot-sdk?

```
// This file is part of Subxt.

// Copyright (C) Parity Technologies (UK) Ltd.

...

```

Reply:
Yup agreed this would be better, though for now I'll just merge this PR to avoid looking into whether it matters at all :)